nestkernel/stimulation_backend_mpi.cpp

```diff
@@ -283,7 +286,21 @@ nest::StimulationBackendMPI::cleanup()
 void
 nest::StimulationBackendMPI::get_port( nest::StimulationDevice* device, std::string* port_name )
 {
-  get_port( device->get_node_id(), device->get_label(), port_name );
+  const std::string& label = device->get_label();
+  // The MPI address can be provided by two different means.
+  // a) the address is given via the mpi_address device status
+  // b) the file is provided via a file: {data_path}/{data_prefix}{label}/{node_id}.txt
+
+  // Case a: MPI address is given via device status, use the supplied address
+  if ( !mpi_address_.empty() )
```

Comment:
Shouldn't this be equivalent?
```suggestion
  if ( mpi_address_ )
```
If not, please use this:
```suggestion
  if ( not mpi_address_.empty() )
```